hail/python/hail/matrixtable.py

```diff
@@ -2533,6 +2533,63 @@ def _localize_entries(self, entries_field_name, cols_field_name) -> 'Table':
         return Table(CastMatrixToTable(
             self._mir, entries_field_name, cols_field_name))
 
+    @typecheck_method(entries_array_field_name=nullable(str),
+                      columns_array_field_name=nullable(str))
+    def localize_entries(self,
+                         entries_array_field_name=None,
+                         columns_array_field_name=None) -> 'Table':
+        """Represent this matrix as a table of entry-rows.
+
+        Examples
+        --------
+        Build a numpy ndarray from a small :class:`.MatrixTable`:
+
+        >>> mt = hl.utils.range_matrix_table(3,3)
+        >>> mt = mt.select_entries(x = mt.row_idx * mt.col_idx)
+        >>> mt.x.show()
+        +---------+---------+-------+
+        | row_idx | col_idx |     x |
+        +---------+---------+-------+
+        |   int32 |   int32 | int32 |
+        +---------+---------+-------+
+        |       0 |       2 |     0 |
+        |       0 |       0 |     0 |
+        |       0 |       1 |     0 |
+        |       1 |       0 |     0 |
+        |       1 |       1 |     1 |
+        |       1 |       2 |     2 |
+        |       2 |       2 |     4 |
+        |       2 |       1 |     2 |
+        |       2 |       0 |     0 |
+        +---------+---------+-------+
+        >>> t = mt.localize_entries('entry_structs', 'columns')
+        >>> t = t.select(entries = t.entry_structs.map(lambda entry: entry.x))
+        >>> np.array(t.entries.collect())
+        array([[0, 0, 0],
+               [0, 1, 2],
+               [0, 2, 4]])
+
```

Comment:
add a note that filtered entries are represented as a missing struct?

Reply:
I ditched the struct on the second to last line so this would blow up without latest numpy, I assume, b/c you can't represent a missing value in a numpy ndarray. I think this is doctested, no?

Reply:
This is the second comment.

Reply:
I still want a notes section with a description of the type!